src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px #f3f3f3 solid;
+  border-radius: 5px;
+
+  &__img {
+    height: 134px;
+    margin: 0 auto;
+    display: block;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+    margin: 40px 0 0;
+  }
+
+  &__code {
+    font-weight: 400;
```

Comment:
Try follow DRY(don't repeat yourself) you can set default font-weight and change it only in place where it is needed